test/foundry/P256Signer.t.sol

```diff
@@ -52,6 +52,7 @@ contract TestP256Signer is Test {
     }
 
     function testFuzzRevertInvalidMessageToSignValidSignatureOld(bytes memory messageToSign_) public {
+        vm.assume(keccak256(messageToSign_) != keccak256(messageToSign));
```

Comment:
waiiiiiiit. WHAT ? :D

Reply:
Yes, the argument passed was `0xcdf841e2c26037e4331174c05ee3e822c5cb6c076b70101637af03284c56e29e` which is the `messageToSign`
<img width="857" alt="Capture d’écran 2023-10-12 à 11 28 17" src="https://github.com/cometh-game/p256-signer/assets/92337658/0e341496-f16b-4bfd-84fc-857a8559fae3">